package.json

```diff
@@ -50,23 +49,18 @@
     "babel-preset-react": "^6.24.1",
     "change-case": "^3.0.2",
     "codecov": "^3.2.0",
-    "commitizen": "^3.0.1",
```

Comment:
We should discuss removing commitizen before going ahead with it.  The team had agreed that they wanted it when the repo was started - this should be a team decision.

Reply:
There are a few reasons for it:
1. We no longer enforce semantic commits, so in its current state it always approves commit messages.
2. It does not work on commits in `master`'s history since we set the commit message when we squash PRs from the Github GUI. The default message there comes from the _PR title_, not any _commit message_.
3. Its latest version from the semver (7.5.2) requires additional configuration. The `yarn.lock` file was the only thing stopping your `git commit`s from exploding.

Reply:
Ah, I misunderstood what commitizen was doing. I readded it for people who want to `yarn commit` instead of `git commit`.